external/storm-rocketmq/src/main/java/org/apache/storm/rocketmq/spout/RocketMqSpout.java

```diff
@@ -60,14 +58,14 @@
 public class RocketMqSpout implements IRichSpout {
     // TODO add metrics
```

Comment:
not only add metrcis, but also print log for MqSpout and MqBolt

Reply:
Thanks. Will update.